Dockerfile.rp-full-dev

```diff
@@ -0,0 +1,23 @@
+FROM registry.access.redhat.com/ubi8/toolbox:8.10
+
+# Needed for installing packages
+USER root 
+
+RUN dnf install https://packages.microsoft.com/config/rhel/8/packages-microsoft-prod.rpm -y \
+    && dnf install make golang azure-cli -y \
```

Comment:
my guess is that we will need to pin `golang` to the specific version in our `go.mod` (in theory this shouldn't be required but we often discover that it is)

Reply:
My team has a nice way of identifying that. See the new changes :)